lib/tasks/create_alignment_config.rake

```diff
@@ -21,6 +21,9 @@ task create_alignment_config: :environment do
     s3_nt_loc_db_path: "#{bucket}/alignment_data/#{name}/nt_loc.#{db_file_ext}",
     s3_nr_db_path: "#{bucket}/alignment_data/#{name}/nr",
     s3_nr_loc_db_path: "#{bucket}/alignment_data/#{name}/nr_loc.#{db_file_ext}",
+    # TODO(mark): Convert nt_info to use BDB.
```

Comment:
It would be nice to start referencing jira tickets in TODOs (https://jira.czi.team/browse/IDSEQ-1536) though that crosses the open and closed source boundary.